spec/test_app_templates/Gemfile.extra

```diff
@@ -4,3 +4,5 @@ group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
 end
+
+gem 'pul_uv_rails', github: 'pulibrary/pul_uv_rails'
```

Comment:
Should  this be released before merging? Otherwise do we need a generator to add this to the Gemfile?

Reply:
@tpendragon @jrgriffiniii @eliotjordan @escowles ?

Reply:
Yes, I believe this will happen today. I'll amend the PR.